pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -646,6 +660,15 @@ func (cm *containerManagerImpl) GetPluginRegistrationHandler() cache.PluginHandl
 // TODO: move the GetResources logic to PodContainerManager.
 func (cm *containerManagerImpl) GetResources(pod *v1.Pod, container *v1.Container) (*kubecontainer.RunContainerOptions, error) {
 	opts := &kubecontainer.RunContainerOptions{}
+	// Set container annotations from the CDI reconciler to
+	// trigger CDI injection
+	if cm.draManager != nil {
+		annotations, err := cm.draManager.GetCDIAnnotations(pod, container)
```

Comment:
With the plan to do dynamic allocation outside of the admission loop, we should look back at how the CPUManager used to allocate CPUs. This also used to be at container start rather than pod admission.